elasticsearch/kibana_dashboards/manageiq_ops_overview.json

```diff
@@ -0,0 +1,13 @@
+{
+  "title": "ManageIQ Ops Overview",
+  "hits": 0,
+  "description": "",
+  "panelsJSON": "[{\"id\":\"hostnames\",\"type\":\"visualization\",\"panelIndex\":1,\"size_x\":7,\"size_y\":4,\"col\":1,\"row\":1}]",
+  "optionsJSON": "{\"darkTheme\":true}",
+  "uiStateJSON": "{}",
+  "version": 1,
+  "timeRestore": false,
+  "kibanaSavedObjectMeta": {
+    "searchSourceJSON": "{\"filter\":[{\"query\":{\"query_string\":{\"query\":\"*\",\"analyze_wildcard\":true}}}]}"
+  }
+}
```

Comment:
This was a work in progress waiting for the actual Dashboard to insert here but due to the problems with actually using/sharing it I never updated here. I will reopen/resubmit when we have a way to share dashboards between users in some manner.